src/content.py

```diff
@@ -28,11 +28,11 @@ def conflicts(base: str, theirs: str, ours: str) -> str | None:
     Returns:
         The description of the merge conflicts or None if there are no conflicts.
     """
-    try:
-        merge(base=base, theirs=theirs, ours=ours)
-    except ContentError as exc:
-        return str(exc)
-    return None
+    # Handle cases that are guaranteed not to have conflicts
+    if theirs in (base, ours) or ours == base:
+        return None
+
+    return f"Detected differences between base, theirs and ours,\ndiff: {diff(theirs, ours)}"
```

Comment:
yeh I thought about it, whether this module should know that it is being executed in that context. For the error messages/ message to the user it would be better I agree, I'll see what to do about it

Reply:
Should be resolved in next commit, removed the explanation since the caller (`check._update_action_problem`) already has an explanatory note. Just leaving this to show the diff